modules/virt-about-vms-and-boot-sources.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-about-vms-and-boot-sources_{context}"]
+
+= About virtual machines and boot sources
+
+Virtual machines consist of a virtual machine definition and one or more disks, backed by data volumes.
+
+Virtual machine templates enable you to create virtual machines using preset virtual machine definitions.
+
+To use the boot sources feature, you must install the latest release of {VirtProductName}. The namespace `openshift-virtualization-os-images` enables the feature and is installed with the {VirtProductName} Operator.
+
+Every virtual machine template requires a boot source, which is a fully configured virtual machine disk image including configured drivers. Each virtual machine template contains a virtual machine definition with a pointer to the boot source. Each boot source has a predefined name and namespace. For many operating systems, a boot source is automatically provided. If it is not provided, then a custom boot source must be prepared by the administrator.
```

Comment:
"For many operating systems" to "In future, for some operating systems

Reply:
We'd have to stick with: "For some operating systems, a boot source is automatically provided" as we can't really reference what might happen in the future, unless this were a marketing doc, of course :-) 

If that's not OK, I can figure something else out. We do provide boot sources for some operating systems today, correct?

Reply:
sgtm, ok to not mention the future :)